app/views/gene.html

```diff
@@ -1,466 +1,693 @@
-      <table class="table geneLinks">
-        <tbody>
-          <tr>
-            <td>cBioPortal link: </td>
-            <td>
-              <a href="https://cbioportal.mskcc.org/ln?q={{::gene.name}}" target="_blank">https://cbioportal.mskcc.org/ln?q={{::gene.name}}</a>
-            </td>
-          </tr>
-          <tr>
-            <td>COSMIC link: </td>
-            <td>
-              <a href="http://cancer.sanger.ac.uk/cosmic/gene/overview?ln={{::gene.name}}" target="_blank">http://cancer.sanger.ac.uk/cosmic/gene/overview?ln={{::gene.name}}</a>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
-  </div>
-  <div id="mutations">
-    <div ng-repeat="mutation in mutations" ng-init="path=getObservePath({type: 'mutation', index: $index});" ng-show="displayCheck(mutation.name_uuid, mutation.name_review)">
-      <div class="sectionStyle">
-        <div ng-style="movingInfo[mutation.name_uuid]" style="display: flex;" class="direct-descendants-margin headerLevel1">
-          <div ng-click="toggleSection(mutation.name_uuid); setSectionOpenStatus('open', [mutation.mutation_effect_uuid])" class="flex-row direct-descendants-margin">
-            <i ng-class="getAngleClass(mutation.name_uuid)"></i>
-              <span>Mutation:</span>
-          </div>
-            <div ng-style="getNameStyle('mutation')" style="flex-grow: 1;" class="flex-row">
-                <realtime-string t="'MUTATION_NAME'" key="'name'" path="path" uuid="getUUID({type: 'mutation', key: 'name', path: path})"></realtime-string>
-                <div ng-show="!reviewMode" class="direct-descendants-margin flex-row">
-                    <span ng-if="mutationContent[mutation.name_uuid].result" ng-bind-html="mutationContent[mutation.name_uuid].result"></span>
-                    <span class="curationNoEntry" ng-if="isEmptySection(mutation, 'mutation')">No Entry</span>
-                    <comments-dict key="'name'" path="path"></comments-dict>
-                </div>
-            </div>
-            <div ng-show="!reviewMode" class="direct-descendants-margin flex-row">
-                  <span ng-show="!reviewMode && me.admin && fileEditable">
-                      <i class="fa fa-arrows" uib-tooltip-placement="top" uib-tooltip="{{movingInfo.message}}" ng-show="displayMoveIcon('mutation', 'initial', mutation.name_uuid)" ng-style="movingInfo.style" ng-click="startMoving('mutation', path, mutation.name_uuid, $index)"></i>
-                      <i class="fa fa-arrows" uib-tooltip-placement="top" uib-tooltip="Click to put above this mutation" ng-show="displayMoveIcon('mutation', 'up', mutation.name_uuid, $index)" style="margin-right:10px;color:green" ng-click="endMoving(path, 'up')"></i>
-                  </span>
-                  <span ng-show="!reviewMode && fileEditable">
-                      <i ng-click="remove('mutation', path)" class="fa fa-trash-o"></i>
-                  </span>
+    <div id="gene" style="margin-bottom:50px">
```

Comment:
Can we set the indention to 2 for html in this pr then later change back to 4 for project upgrade? Just want to make sure no unexpected changes made through.

Reply:
Sure

Reply:
Sorry, I tried to reformat code but the style totally messed up. Could you click `ignore space` in review mode?